Week3/homework/step2-7.js

```diff
@@ -20,4 +20,13 @@ f2(y);
 
 console.log(y);
 
-// Add your explanation as a comment here
+/* In javascript it's always pass by reference.when a variable refers to an object which includes array, 
```

Comment:
👍 but I think there is a typo - In javascript it's always pass by **value**. When a variable refers to an object, the value is the reference to the object.